src/toolbar.js

```diff
@@ -204,6 +216,11 @@ Toolbar.prototype.addDefaultLinkButton = function(label) {
 
         // Add link to selection
         document.execCommand('createLink', false, me.urlTextbox.value);
+
+        if (options.openInNewTab) {
+          // Add target='_blank' to open in a new tab
+          selection.getParentElement().target = '_blank';
```

Comment:
This makes an assumption about where our selection is after the `execCommand` that I don't think we can necessarily make.